test/test_visual_utils.py

```diff
@@ -15,15 +15,15 @@ def test_observations_to_image():
         "depth": np.random.rand(200, 400, 1),
     }
     info = {
-        "collisions.is_collision": True,
-        "top_down_map.map": np.random.randint(
-            low=0, high=255, size=(300, 300)
-        ),
-        "top_down_map.fog_of_war_mask": np.random.randint(
-            low=0, high=1, size=(300, 300)
-        ),
-        "top_down_map.agent_map_coord": [(10, 10)],
-        "top_down_map.agent_angle": [np.random.random()],
+        "collisions": {"is_collision": True},
+        "top_down_map": {
+            "map": np.random.randint(low=0, high=255, size=(300, 300)),
+            "fog_of_war_mask": np.random.randint(
+                low=0, high=1, size=(300, 300)
+            ),
+            "agent_map_coord": [(10, 10)],
+            "agent_angle": [np.random.random()],
+        },
```

Comment:
Why are we unflattening the info dict? We do need it flatten so we can report nested scalar values in tensorboard and wandb correctly/